coreos-base/update-ssh-keys/update-ssh-keys-9999.ebuild

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2017 CoreOS, Inc.. All rights reserved.
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
Do these changes want revision bumps for clarity?

Reply:
I had kept the versions since the package contents should not have changed (unless we had some different settings between the host and cross-compiler linkers or something like that), but they're updated now.